src/DynamoUtilities/DataMarshaler.cs

```diff
@@ -76,14 +76,6 @@ public object Marshal(object obj)
 
             var targetType = obj.GetType();
 
-            // TODO: Remove this conversion after updating IronPython version in 2.1
-            // in which IronPython will support Int64
-            if (typeof (long) == targetType)
```

Comment:
was there any change to the output marsheller?

Reply:
Not that I am aware of but @aparajit-pratap may have more knowledge here.  This was the original change I believe:

https://github.com/DynamoDS/Dynamo/pull/8602/files